### ISSUE TITLE
silence preprocessor redefinition warnings

### ISSUE THREAD

**Issue by slayoo** [tvatter/vinecopulib#28]
```
In file included from /home/vagrant/devel/vinecopulib/src/../include/bicop_class.hpp:38:0,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_parametric.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_archimedean.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/bicop_archimedean.cpp:20:
/home/vagrant/devel/vinecopulib/src/../include/c_tools.h:8:0: warning: "false" redefined
 #define false 0
 ^
In file included from /usr/include/c++/4.9/bits/atomic_base.h:36:0,
                 from /usr/include/c++/4.9/atomic:41,
                 from /usr/include/boost/math/special_functions/detail/bernoulli_details.hpp:19,
                 from /usr/include/boost/math/special_functions/bernoulli.hpp:16,
                 from /usr/include/boost/math/special_functions/gamma.hpp:35,
                 from /home/vagrant/devel/vinecopulib/src/../include/boost_tools.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_class.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_parametric.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_archimedean.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/bicop_archimedean.cpp:20:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/stdbool.h:42:0: note: this is the location of the previous definition
 #define false false
 ^
In file included from /home/vagrant/devel/vinecopulib/src/../include/bicop_class.hpp:38:0,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_parametric.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_archimedean.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/bicop_archimedean.cpp:20:
/home/vagrant/devel/vinecopulib/src/../include/c_tools.h:9:0: warning: "true" redefined
 #define true (!false)
 ^
In file included from /usr/include/c++/4.9/bits/atomic_base.h:36:0,
                 from /usr/include/c++/4.9/atomic:41,
                 from /usr/include/boost/math/special_functions/detail/bernoulli_details.hpp:19,
                 from /usr/include/boost/math/special_functions/bernoulli.hpp:16,
                 from /usr/include/boost/math/special_functions/gamma.hpp:35,
                 from /home/vagrant/devel/vinecopulib/src/../include/boost_tools.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_class.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_parametric.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/../include/bicop_archimedean.hpp:23,
                 from /home/vagrant/devel/vinecopulib/src/bicop_archimedean.cpp:20:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/stdbool.h:43:0: note: this is the location of the previous definition
 #define true true
```

**Comment by tvatter** [comment]
Where are we on this?

**Comment by tnagler** [comment]
do we actually need to define true/false there?

**Comment by tvatter** [comment]
Probably not, we'll modify in the clean-up.

**Comment by tnagler** [comment]
fixed by 146087fd7c7c76a55fc22e764e82537cb26a5e98
